modules/ROOT/pages/whats-new-in-studio.adoc

```diff
@@ -1,31 +1,35 @@
 = What’s New in Studio
 
-Anypoint Studio 7.12 introduces support for comprehensive publication of assets to Exchange and for multi-factor authentication (MFA) to Anypoint Platform.
+Anypoint Studio 7.13 introduces support for message flow visualization and management of DataWeave libraries through its UI.
```

Comment:
through its UI > through the UI

Reply:
If using "the", it would be necessary to specify which UI we are talking about, so that's why I decided to use the "its", since it is clearer for the reader that the UI belongs to Studio.